src/index.html

```diff
@@ -11,7 +11,174 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      onsubmit="onSubmit()"
+    >
+      <fieldset class="form_fieldSet">
+        <legend>Personal information</legend>
+        <div class="form_field">
+          <label for="Surname">Surname:</label>
+          <input
+            type="text"
+            name="Surname"
+            id="Surname"
+          ></input>
+        </div>
+        <div class="form_field">
+          <label for="Name">Name:</label>
+          <input
+            type="text"
+            name="Name"
+            id="Name"
+          ></input>
+        </div>
+        <div class="form_field">
+          <label for="Age">How old are You?</label>
+          <input
+            type="number"
+            name="Age"
+            min="12"
+            max="100"
+            value="12"
+            id="Age"
+          ></input>
+        </div>
+        <div class="form_field">
+          <label for="BirthDate">Full date of birth:</label>
+          <input
+            type="date"
+            name="BirthDate"
+            id="BirthDate"
+          ></input>
+        </div>
+        <div class="form_field">
+          <label for="Terms">I accept the term of the agreement</label>
+          <input
+            type="checkbox"
+            name="Terms"
+            required
+            id="Terms"
+          ></input>
+        </div>
+      </fieldset>
+      <fieldset class="form_fieldSet">
+        <legend>Registration</legend>
+        <div class="form_field">
+          <label for="E-mail">E-mail:</label>
+          <input
+            type="email"
+            name="E-mail"
+            required
+            placeholder="email@example.com"
+            id="E-mail"
+          ></input>
+        </div>
+        <div class="form_field">
+          <label for="Password">Password:</label>
+          <input
+            type="password"
+            name="Password"
+            required
+            id="Password"
+          ></input>
+        </div>
+      </fieldset>
+      <fieldset class="form_fieldSet">
+        <legend>An interesting fact about you!</legend>
+        <div class="form_field">
+          Do you love cats?
+          <label for="Agree">
+            <input type="radio"
+              name="lc"
```

Comment:
Do not shorten your variables
```suggestion
              name="loveCats"
```